File: 백준/Bronze/2920. 음계/음계.cc
#include<iostream>
using namespace std;

int main(){
	int arr[8]{0,};

	for (int i = 0;i<8;i++) {
		cin >> arr[i];
	}

	if (arr[0]==1) {
		for (int i = 0;i < 8;i++) {
			if (arr[i] != i + 1) {
				cout << "mixed";
				return 0;
			}
		}

		cout << "ascending";
		return 0;

	}
	else if (arr[0] == 8) {
		for (int i = 0;i < 8;i++) {
			if (arr[i] != 8-i) {
				cout << "mixed";
				return 0;
			}
		}
		cout << "descending";
		return 0;
	}
	else {
		cout << "mixed";
		return 0;
	}
}

File: 백준/Bronze/2920. 음계/음계.py
import sys

# map(int,sys.stdin.readline().split())
# data = list(map(int,sys.stdin.readline().split()))

li = list(map(int,sys.stdin.readline().split()))

asd = [1,2,3,4,5,6,7,8]

if li == asd:
    print("ascending")
else:
    asd.reverse()
    if li ==asd:
        print("descending")
    else:
        print("mixed")